### DISCUSSION TITLE
Community Project: Extract audio effects

### DISCUSSION THREAD

**Discussion by SullyPanda76cl** [AssetRipper/AssetRipper#1585] [Q&A]
Hi.. I've been using this tool and is fantastic...
I've exported lots of PNG, and fonts....

I've haven't been able to find the game sounds (gun firing, hitting a foe, gui click, etc)
Has anyone any knowledge if there's a trick for that?

In particular, I'm using it with Steams' "Lust Goddess"

![image](https://github.com/user-attachments/assets/82ff8ede-b492-44f6-9b6b-a5fea13863d5)

**Comment by ds5678** [comment]
Took a quick look on steamdb. The game directory seems a little sparse. Maybe they're downloaded to another location?
